packages/react-radio/src/components/Radio/useRadioStyles.ts

```diff
@@ -1,11 +1,19 @@
 import { createFocusOutlineStyle } from '@fluentui/react-tabster';
 import { tokens } from '@fluentui/react-theme';
 import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
-import type { RadioState } from './Radio.types';
+import type { RadioSlots, RadioState } from './Radio.types';
+import type { SlotClassNames } from '@fluentui/react-utilities';
 
+/**
+ * @deprecated
+ */
 export const radioClassName = 'fui-Radio';
```

Comment:
For `Radio` you can actually remove this class name instead of deprecating it since the component is still under `unstable`:

```suggestion
```